starter-code/scripts/models/article.js

```diff
@@ -61,17 +62,19 @@ Article.fetchAll = function() {
       1.a Load our json data
       1.b Store that data in localStorage so that we can skip the server call next time,
       1.c And then render the index page.*/
-      $.getJSON('../../data/hackerIpsum.json1').then(
+      $.getJSON('../../data/hackerIpsum.json').then(
+        //parses data for us
       function(data) {
-        console.log(data);
+        localStorage.setItem('hackerIpsum',JSON.stringify(data));
+        //localStorage only takes JSON
+        Article.loadAll(data);
+        articleView.renderIndexPage();
```

Comment:
Nice work ensuring that your data is available, storing it in local storage, constructing our article objects and the array of articles, and then rendering the page :D